examples/key_map/main.ml

```diff
@@ -0,0 +1,48 @@
+open Leaves
+open Minttea
+
+type msg = CursorUp | CursorDown | CursorLeft
+
+let defaults =
+  let open Key_map in
+  make [
+    on
+      ~help:{ key = "up"; desc = "↑/k" }
+      [ Minttea.Event.Up; Minttea.Event.Key "k" ]
+      CursorUp;
+    on
+      ~help:{ key = "down"; desc = "↓/j" }
```

Comment:
What do you think about deriving these from the events and using the `~help` for writing more about what the keymap is supposed to do?

For example:

```ocaml
let keymap = Key_map.make [
  on Event.[Up; Key "k"] CursorUp "moves the cursor up";
  on Event.[Key "q"] Quit "quit the app";
]
```

Reply:
I can see how with this information we can put together a bindings help view:

```
↑/k: moves the cursor up
q: quit the app
```